junit-platform-commons/src/main/java/org/junit/platform/commons/util/AnnotationUtils.java

```diff
@@ -258,6 +258,10 @@ else if (candidate.annotationType().equals(containerType)) {
 		// @formatter:on
 	}
 
+	public static List<Method> findAnnotatedMethods(Class<?> clazz, Class<? extends Annotation> annotationType) {
+		return findAnnotatedMethods(clazz, annotationType, MethodSortOrder.HierarchyDown);
+	}
+
```

Comment:
I don't think we should set a default for`MethodSortOrder`. That's the caller's responsibility and may lead to subtle bugs. So, I think we should remove this method.